src/System.IO.FileSystem/src/System/IO/Enumeration/FileSystemEnumerator.Unix.cs

```diff
@@ -67,18 +65,16 @@ public FileSystemEnumerator(string directory, EnumerationOptions options = null)
             }
         }
 
-        private SafeDirectoryHandle CreateDirectoryHandle(string path)
+        private IntPtr CreateDirectoryHandle(string path)
```

Comment:
Call sites to this compare it to null. Aren't those broken now?

Reply:
Missed the one. Fixing.

Reply:
Are we missing tests then, too?

Reply:
Hmm... I'll think about how to test this. It is an odd case.

Reply:
Thanks